test/remote/gateways/remote_rapyd_test.rb

```diff
@@ -146,7 +146,7 @@ def test_partial_refund
   def test_failed_refund
     response = @gateway.refund(@amount, '')
     assert_failure response
-    assert_equal 'The request tried to retrieve a payment, but the payment was not found. The request was rejected. Corrective action: Use a valid payment ID.', response.message
+    assert_equal 'The request attempted an operation that requires a payment ID, but the payment was not found. The request was rejected. Corrective action: Use the ID of a valid payment.', response.message
```

Comment:
The difference in response is because of the change I made to the conditional in `add_payment`, where it is attempting to send a `null` payment instead of `''` (empty string).

Reply:
Thanks for this explanation 👍